app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -563,6 +563,15 @@ class EditPatientScreen : BaseScreen<
     }
   }
 
+  private fun highlightNewlyScannedBpPassports(bpPassportView: PatientEditBpPassportViewBinding) {
+    bpPassportView.bpPassportIdentifier.setBackgroundColor(resources.getColor(R.color.simple_yellow_100))
```

Comment:
Can we use `context.resolveColor` instead, please? `resources.getColor` is deprecated.